package.json

```diff
@@ -13,14 +13,15 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@patternfly/patternfly": "4.164.2",
-    "@patternfly/react-core": "4.157.3",
-    "@patternfly/react-icons": "4.11.17",
-    "@patternfly/react-styles": "4.11.16",
-    "@patternfly/react-table": "4.30.3",
-    "@rhoas/app-services-ui-shared": "^0.15.4",
+    "@patternfly/react-core": "4.181.1",
+    "@patternfly/react-icons": "4.32.1",
+    "@patternfly/react-styles": "4.31.1",
+    "@patternfly/react-table": "4.50.1",
+    "@rhoas/app-services-ui-components": "1.11.8",
```

Comment:
https://www.npmjs.com/package/@rhoas/app-services-ui-components?activeTab=readme says it is not recommended install as direct dependency. Do we meet this recommendation? Sorry if that is a silly question. I have more experience with testing by cypress than development in react. 

Also the latest seems to be a little bit higher `1.24.1`. Is the current version expected?

Reply:
> https://www.npmjs.com/package/@rhoas/app-services-ui-components?activeTab=readme says it is not recommended install as direct dependency. Do we meet this recommendation? Sorry if that is a silly question. I have more experience with testing by cypress than development in react.

It's not a silly question at all. Actually it's an important aspect.
I think the recommendation you linked is not necessary.
By declaring dependencies as shared ([here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/main/webpack.common.js#L130)), they are loaded at runtime only if the host application hasn't already done it.
In this way the same dependency is loaded only once by the first module that requests it.
Given that, I'd say it's not necessary to rely to `peerDependencies`. If the shared dep version matches an existing dep of the host application, that's enough.

> 
> Also the latest seems to be a little bit higher `1.24.1`. Is the current version expected?

I'm aligning with other repos here. They are all using this version of PF. So if we want to reuse it without further dependency loading, we have to stick to that. 
We could argue if it make sense to use fixed version numbers instead of `^version`, but that's another discussion :)